docs/2_registration_project_access_session.md

```diff
@@ -174,31 +174,29 @@ dds data get
 
 Where spf.txt is the path to a file you have created with the following format:
 
->cat spf.txt
 ~~~
 data/example_directory_1/sub_directory_1/
 data/example_directory_2/sub_directory_2/example_file_3.txt
 data/example_directory_2/example_file_5.txt
 ~~~
 
-2. In reality, the data you've uploaded will be downloaded by researchers.To [invite researchers](https://scilifelabdatacentre.github.io/dds_cli/project/#dds-project-access-grant), you can use the commands:
+ - [ ] TASK: Release the project after making sure all the data was uploaded, remember to specify the correct project ID
+ - [ ] TASK: Download the project contents, and verify that they are the correct. You can download data with your unit admin account.
 
-~~~
-dds project access grant --project "<Project ID>" --email "<email>"
-~~~
+#### Researchers
```

Comment:
I prefer having one complete file with all the instructions rather than several small ones. 
I am having trouble understanding our goal on this because I think everyone has an idea. 

Currently, we only have one slide related to the demo, and it is before explaining project life cycle. Are we doing several demos, one of each file? One big demo at the end? I am quite confused at this point

Reply:
Let's get the small change suggestions done without moving things around in this PR.

Reply:
So. Do I merged this two requests then?

Reply:
If everything except the more major 'moving some content to a different file' is done, then I would say yes.